### ISSUE TITLE
Add ScheduleInvocableType to interface 

### ISSUE THREAD

**Issue by lzinga** [jamesmh/coravel#92]
**Describe the solution you'd like**
The ability to add a scheduled IInvocable through a type instead of through generic, action, or func. There already seems to be a method for this named [IScheduleInterval ScheduleInvocableType(Type invocableType)](https://github.com/jamesmh/coravel/blob/master/Src/Coravel/Scheduling/Schedule/Scheduler.cs#L62-L72) but it is not a requirement from the interface of `IScheduler`.

**Describe alternatives you've considered**
You could work around and schedule a list of types like below but it would not work nicely with dependency injection since nothing would actually be injected from the service provider.
```csharp
foreach(var type in types)
{
    schedule.Schedule(() =>
    {
        (Activator.CreateInstance(type) as IInvocable).Invoke();
    }).Cron("*/5 * * * *");
}
```
I did fork and make the changes [here](https://github.com/lzinga/coravel/commit/53af00c59335c74f2867d05c5b5da0ba8a82c7b6). From the looks of it no tests or anything would really need to be added since the tests are already using the method in question. This just allows it to be used with the IScheduler. Though if I am missing something that already does this it would be great to know!

**How it would appear**
If the other one was available through the interface we could do - 
```csharp
foreach(var type in types)
{
    schedule.ScheduleInvocableType(type).Cron("*/5 * * * *");
}
```

**Comment by jamesmh** [comment]
I'll look at adding this soon 👌

**Comment by jamesmh** [comment]
Should be available in 2.6.0 now.
